pkgs/development/python-modules/argcomplete/default.nix

```diff
@@ -21,6 +21,13 @@ buildPythonPackage rec {
     hash = "sha256-um8iFzEHExTRV1BAl86/XKLc7vmf2Ws1dB83agfvoec=";
   };
 
+  patches = [
+    # fixes issues with python3Packages.traitlets tests
+    # https://git.launchpad.net/ubuntu/+source/python-argcomplete/tree/debian/patches/python-3.13-compat.patch?h=ubuntu/plucky
+    # https://github.com/kislyuk/argcomplete/pull/513
+    ./python-3.13-compat.patch
```

Comment:
Cant we use the Github patch link directly instead of reuploading it herer again?

https://github.com/kislyuk/argcomplete/commit/7438d1fa962eb736af9754669f200f29c5b6025d.patch

Reply:
That commit is not merged yet, it may disappear after a force push.

Reply:
the pr is merged upstream

Reply:
It was reverted later via force-push to `develop`.